package.json

```diff
@@ -18,8 +18,9 @@
 	},
 	"scripts": {
 		"watch": "yarn install-if-deps-outdated && ./node_modules/.bin/gulp watch",
-		"clean": "yarn clean-client && yarn clean-extensions",
+		"clean": "yarn clean-client && yarn clean-extensions && yarn clean-composer",
 		"clean-client": "rm -rf _inc/build/ css/",
+		"clean-composer": "rm -rf composer.lock vendor/",
```

Comment:
Nit: Considering the fact that this feature should have existed in `composer` out of the box already, I expect it to be a composer script instead - invokable by `composer clean` for example. We already use `composer phpunit` for running the in-package unit tests. What do you think?

Reply:
I would prefer to stay consistent with our existing naming conventions for cleaning rather than introduce new ones because we wish they existed in composer itself.

Reply:
Sounds good to me 👍